src/codeu/chat/common/SQLFormatter.java

```diff
@@ -0,0 +1,106 @@
+package codeu.chat.common;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import codeu.chat.util.Uuid;
+import codeu.chat.util.Time;
+
+/**
+ * Created by strobe on 4/04/17.
+ */
+public class SQLFormatter {
+    public static String sqlID(Uuid userID){
+        String sqlID = "NULL";
+        if(userID != null) {
+            sqlID = userID.toString();
+            sqlID = sqlID.replace("[UUID:","");
+            sqlID = sqlID.replace("]","");
+            sqlID = "'" + sqlID + "'";
+        }
+
+        return sqlID;
+    }
+
+    public static String sqlID(Uuid userID1, Uuid userID2){
+        String sqlID1 = "NULL";
+        if(userID1 != null) {
+            sqlID1 = userID1.toString();
+            sqlID1 = sqlID1.replace("[UUID:","");
+            sqlID1 = sqlID1.replace("]","");
+        }
+
+        String sqlID2 = "NULL";
+        if(userID2 != null) {
+            sqlID2 = userID2.toString();
+            sqlID2 = sqlID2.replace("[UUID:","");
+            sqlID2 = sqlID2.replace("]","");
+        }
+
+        String sqlID = "'" + sqlID1 + sqlID2 + "'";
+        return sqlID;
+    }
+
+    public static String sqlName(String userName){
+        String sqlName = "'" + userName + "'";
```

Comment:
What happens if a user creates a username with a ' ? I think SQL injection might be possible here and in any other case where a string from the user is provided.